src/test/java/jenkins/scm/impl/mock/MockSCMNavigator.java

```diff
@@ -52,6 +46,13 @@
 import org.jenkinsci.Symbol;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+import javax.annotation.Nonnull;
```

Comment:
Where is the published JSR defining a spec for a `javax` annotation of this name?